pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -406,6 +406,9 @@ func (a *ofLearnAction) MatchTransportDst(protocol Protocol) LearnAction {
 	// generate the field name.
 	fieldName := fmt.Sprintf("OXM_OF_%s_DST", strings.ToUpper(string(protocol)))
 	a.nxLearn.AddMatch(&ofctrl.LearnField{Name: fieldName}, 2*8, &ofctrl.LearnField{Name: fieldName}, nil)
+	if protocol == ProtocolTCPv6 {
+		a.nxLearn.AddMatch(&ofctrl.LearnField{Name: "NXM_OF_TCP_DST"}, 2*8, &ofctrl.LearnField{Name: "NXM_OF_TCP_DST"}, nil)
```

Comment:
OXM doesn't support TCPv6?
don't we need to handle UDPv6?

Reply:
Well, the original logic doesn't work but "NXM_OF_TCP_DST" works so I thought OXM isn't for IPv6. But actually the problem in original logic is that there is no field called "OXM_OF_TCPV6_DST".
Now, it is guaranteed that no `v6` exists in OXM field.